### PR TITLE
Rm/climber

### PR DESCRIPTION
**Pull request by PrathamRawat**

### CODE REVIEW

File: src/main/java/frc/robot/subsystems/Climber.java [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    private CANSparkMax liftMotor;
+    private CANSparkMax yoyoMotor;
+
+    public Climber() {
+        liftMotor = new CANSparkMax(Constants.CLIMBER_LIFT_MOTOR_PORT, MotorType.kBrushless);
+        yoyoMotor = new CANSparkMax(Constants.CLIMBER_YOYO_MOTOR_PORT, MotorType.kBrushless);
+    }
+
+    public void climbUp() {
+        liftMotor.set(1.0);
```
Comment (PrathamRawat) [inline src/main/java/frc/robot/subsystems/Climber.java]: The speeds for climbUp and climbDown should be a constant that we can potentially change if needed

File: src/main/java/frc/robot/subsystems/Funnel.java [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+public class Funnel extends SubsystemBase {
+
+    private CANSparkMax motor;
+
+    public Funnel() {
+        motor = new CANSparkMax(Constants.FUNNEL_MOTOR_PORT, MotorType.kBrushless);
+    }
+
+    public void funnel() {
+        motor.set(1.0);
```
Comment (PrathamRawat) [inline src/main/java/frc/robot/subsystems/Funnel.java]: Should also make constants for funneling speeds

**Review by PrathamRawat — COMMENTED** [review]
Changes not approved

**Review by PrathamRawat — COMMENTED** [review]
Fix merge conflicts

**Review by rmui1 — APPROVED** [review]

**Pull request merged**